packages/structure/src/model/RWRoute.ts

```diff
@@ -141,8 +141,27 @@ export class RWRoute extends BaseNode {
   }
 
   @lazy() get prerender(): boolean {
-    return this.getBoolAttr('prerender')
+    // Note, using this.getBooleanAttr always returns true, unsure why.
```

Comment:
Did you consider fixing `this.getBoolAttr` instead?

https://s.tape.sh/NmKMGZBt